app/javascript/test/game.test.js

```diff
@@ -1,24 +1,53 @@
 import React from 'react';
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+import { mount } from 'enzyme';
+
 import './setupTests';
```

Comment:
You should be able to override the test command in`package.json` so that you don't have to import `./setupTests` in each test file.

```js
"scripts": {
  "test": "jest --setupFiles ./app/javascript/setupTests.js"
},
```